core/api/src/main/java/org/eclipse/sensinact/core/twin/SensinactDigitalTwin.java

```diff
@@ -72,14 +81,24 @@ public interface SensinactDigitalTwin extends CommandScoped {
      */
     SensinactProvider createProvider(String model, String providerName);
 
+    /**
+     * Create a provider instance for the named model
+     *
+     * @param modelPackageUri
+     * @param model
+     * @param providerName
+     * @return
+     */
+    SensinactProvider createProvider(String modelPackageUri, String model, String providerName);
+
     /**
      * Create a provider instance for the named model
      *
      * @param model
      * @param providerName
      * @return
      */
-    SensinactProvider createProvider(String model, String providerName, Instant created);
```

Comment:
I think this should be an additional method not a replacement?

Reply:
done,